src/components/inputs/TextField.js

```diff
@@ -473,10 +473,13 @@ export default class TextField extends BaseInput {
       color,
       ...others
     } = this.getThemeProps();
+
     const typography = this.getTypography();
     const {lineHeight, ...typographyStyle} = typography;
     const textColor = this.getStateColor(color || this.extractColorValue());
     const hasRightElement = this.shouldDisplayRightButton() || rightIconSource;
+    const shouldUseMultiline = multiline ? multiline : expandable;
```

Comment:
a ternary when one of the values is the condition can be changed to pipes ->  `multiline || expandable`

Reply:
Done.